src/testdrive/src/action.rs

```diff
@@ -56,34 +55,6 @@ pub struct Config {
     pub reset_materialized: bool,
 }
 
-impl Default for Config {
```

Comment:
This wasn't previously dead code, actually! It was the source of truth for config defaults. But I wanted to use structopts built-in support for default values, so it was easier to get rid of this impl and just say you only get defaults if you use testdrive's CLI, and not its library interface.

And FWIW it can't be `pub(crate)` because it's part of testdrive's public library API:  https://mtrlz.dev/api/rust/testdrive/index.html